frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -54,7 +54,7 @@ export const ExpressionWidget = (props: ExpressionWidgetProps) => {
 
   const [name, setName] = useState(initialName || "");
   const [clause, setClause] = useState<Lib.ExpressionClause | null>(
-    initialClause ?? null,
+    (initialClause ?? null) as Lib.ExpressionClause | null,
```

Comment:
@ranquild we need to cast the `Expressionable` at some point to be able to pass it back to `onChangeClause` as a `Lib.ExpressionClause`.